docker-compose/monitor/README.md

```diff
@@ -83,6 +85,40 @@ Then navigate to the Monitor tab at http://localhost:16686/monitor to view the R
 
 ![My Service RED Metrics](images/my_service_metrics.png)
 
+## Migrating to Span Metrics Connector 
+
+### Background
+
+A new [Connector](https://pkg.go.dev/go.opentelemetry.io/collector/connector#section-readme) component was introduced
+to the OpenTelemetry Collector to provide a means of receiving and exporting between any type of telemetry.
+
+The existing [Span Metrics Processor][spanmetricsprocessor] was a good candidate to migrate over to the connector type,
+resulting in the new [Span Metrics Connector][spanmetricsconnector] component.
+
+The Span Metrics Connector variant introduces some [breaking changes][processor-to-connector], and the following
+section aims to provide the instructions necessary to use the metrics produced by this component.
+
+### Migrating
+
+Assuming the OpenTelemetry Collector is running with the [Span Metrics Connector][spanmetricsconnector] correctly
+configured, the following configuration should be applied to jaeger-query or jaeger-all-in-one:
+
+As command line parameters:
+```shell
+--prometheus.query.namespace=span_metrics
+--prometheus.query.duration-metric-name=duration
+--prometheus.query.duration-unit=ms
+--prometheus.query.span-name-label=span_name
```

Comment:
I'm skeptical about introducing this level of flexibility, is it really that necessary? It adds both code complexity and mental complexity for the users. To support connector we could just expose a single flag and deal with the renaming internally.

Reply:
1) I as thinking just a Boolean, default to false for backwards compat, in 2 releases default to true, in next two releases remove altogether.

2) is the dependency on the naming directly in spm or in prom storage impl?

Reply:
> I as thinking just a Boolean, default to false for backwards compat, in 2 releases default to true, in next two releases remove altogether.

Sounds good. 👍🏼 

> is the dependency on the naming directly in spm or in prom storage impl?

The name changes are in the metrics stored in Prometheys themselves. Example:
- Before (spanmetrics processor): `latency_bucket{service_name="foo", operation="GetCustomer", ...}`
- After (spanmetrics connector): `my_namespace_duration_milliseconds_bucket{service_name="foo", span_name="GetCustomer", ...}`

So main changes I've made are in the Prometheus reader in Jaeger, in how it puts together the Prometheus query.

Reply:
My initial thinking was that having this boolean parameter under the prometheus namespace doesn't seem quite right because it relates to OTEL and the data contract it has with Jaeger via the SPM feature; which is why I felt having it under an `spm` namespace made more sense.

But given it's going to be removed after 2 releases, maybe it's not worth the extra effort to create a new namespace?

Reply:
The reason I asked was that it seems even as Boolean it is scoped to prom implementation, not the top level spm.

Reply:
Sorry, do you mean you prefer a boolean under the `prometheus` scope? I don't have a strong opinion either way, especially given it will be temporary.

By the way, if we go down this path of a boolean parameter, there are some new parameters that are still necessary to:
- Configure the namespace.
- Configure the unit used for latency histograms.

Those parameters are also configurable in the spanmetrics connector and influence the metric's name/labels. So those will need to remain.

Reply:
> do you mean you prefer a boolean under the prometheus scope?

yes

Reply:
I've added a `prometheus.query.support-spanmetrics-connector` flag in: [e52b74b](https://github.com/jaegertracing/jaeger/pull/4452/commits/e52b74b63478c3a09d00dc98ba003b95972f2b67)

Let me know what you think.